backends/occa/ceed-occa-qfunction.c

```diff
@@ -134,7 +134,13 @@ static int CeedQFunctionApply_Occa(CeedQFunction qf, CeedInt Q,
   // ***************************************************************************
   void *ctx;
   if (cbytes>0) {
-    ierr = CeedQFunctionGetContext(qf, &ctx); CeedChk(ierr);
+    bool fortranstatus;
```

Comment:
If the backend does not use the Fortran interface to access the Fortran version user QFunction, then the backend is responsible for making a special call to get the true context data. This would apply to OCCA and CUDA.